### PR TITLE
infoschema: fix show information_schema.table default value bug and refine metric table comment. 

### PR DESCRIPTION
**Pull request by crazycs520**
### What problem does this PR solve? 
* Fix default value of `show create table` in system tables;
before this PR
```sql
metrics_schema> show create table tidb_query_duration;
+---------------------+--------------------------------------------------------------------------------------------------------------------+
| Table               | Create Table                                                                                                       |
+---------------------+--------------------------------------------------------------------------------------------------------------------+
| tidb_query_duration | CREATE TABLE `tidb_query_duration` (                                                                               |
|                     |   `time` datetime unsigned DEFAULT NULL,                                                                           |
|                     |   `instance` varchar(512) DEFAULT NULL,                                                                            |
|                     |   `sql_type` varchar(512) DEFAULT NULL,                                                                            |
|                     |   `quantile` double unsigned DEFAULT NULL,                                                                         |
|                     |   `value` double unsigned DEFAULT NULL                                                                             |
|                     | ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin COMMENT='The quantile of TiDB query durations(second)' |
+---------------------+--------------------------------------------------------------------------------------------------------------------+
```

This PR

```sql
metrics_schema> show create table tidb_query_duration;
+---------------------+--------------------------------------------------------------------------------------------------------------------+
| Table               | Create Table                                                                                                       |
+---------------------+--------------------------------------------------------------------------------------------------------------------+
| tidb_query_duration | CREATE TABLE `tidb_query_duration` (                                                                               |
|                     |   `time` datetime unsigned DEFAULT CURRENT_TIMESTAMP,                                                              |
|                     |   `instance` varchar(512) DEFAULT NULL,                                                                            |
|                     |   `sql_type` varchar(512) DEFAULT NULL,                                                                            |
|                     |   `quantile` double unsigned DEFAULT '0.9',                                                                        |
|                     |   `value` double unsigned DEFAULT NULL                                                                             |
|                     | ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin COMMENT='The quantile of TiDB query durations(second)' |
+---------------------+--------------------------------------------------------------------------------------------------------------------+
```

* refine some metric-table comment and name.
* update go.mod for https://github.com/pingcap/sysutil/pull/12

### Check List 

Tests 

 - Unit test

Related changes

 - No

### CODE REVIEW

File: executor/inspection_summary.go [hunk 1]
```diff
@@ -168,11 +168,11 @@ var inspectionSummaryRules = map[string][]string{
 		"tikv_cop_request_duration",
 		"tikv_cop_request_durations",
 		"tikv_cop_scan_details",
-		"tikv_cop_total_dag_executors",
```
Comment (crazycs520) [inline executor/inspection_summary.go]: The original name is not accuracy.

**Comment by reafans** [comment]
lgtm

**Pull request merged**